.github/ISSUE_TEMPLATE/enhancement.md

```diff
@@ -0,0 +1,18 @@
+---
+name: Enhancement Request
```

Comment:
What is the difference between an enhancement and a feature request?

Reply:
it's the same actually, removed feature request, could refer to https://github.com/kubernetes/kubernetes/issues/new/choose